MaxText/layers.py

```diff
@@ -222,6 +223,8 @@ def __call__(self, inputs: Array) -> Array:
 
     contract_ind = tuple(range(0, len(axis)))
 
+    # quantization - maxtext specific? for short term purposes, bf16 should be
+    # sufficient?
```

Comment:
We are currently performing int8 experiments now to see the potential benefits. All perf numbers are based on bf16 - I think its okay if pytorch doesn't support int8 to start, but at some point it should since its likely there is a large performance benefit.